test/cli_search_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/util"
+	"github.com/gotestyourself/gotestyourself/icmd"
+
+	"github.com/go-check/check"
+)
+
+// PouchSearchSuite is the test suite for search CLI.
+type PouchSearchSuite struct{}
+
+func init() {
+	check.Suite(&PouchSearchSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchSearchSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TestSearchWorks tests "pouch search" work.
+func (suite *PouchSearchSuite) TestSearchWorks(c *check.C) {
+	res := command.PouchRun("search", "nginx")
+	res.Assert(c, icmd.Success)
+}
+
+// TestSearchInSpecificRegistry test search in specific registry
+func (suite *PouchSearchSuite) TestSearchInSpecificRegistry(c *check.C) {
```

Comment:
I think we can put all testcases into one func `TestSearchWorks`, wdyt?

Reply:
you are right